languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -51106,6 +51112,103 @@ though, and has since been slightly modified)-->
+                <token>es</token>
+                <token>t</token>
+            </antipattern>
+            <pattern>
+                <token regexp="yes">quel(le)?s?<exception scope="previous" regexp="yes">tel(le)?s?</exception></token>
+                <marker>
+                    <token regexp="yes">es|ai|aie|aies|et<exception scope="next" postag="A.*" postag_regexp="yes"/></token>
+                </marker>
+            </pattern>
+            <message>Un verbe est attendu après l'adjectif interrogatif '\1'.</message>
+            <suggestion>est</suggestion>
+            <example correction="est">Quelle <marker>et</marker> la plus grande des maisons habitables ?</example>
+            <example>Beaucoup de firmes multinationales allemandes sont originaires de la Bavière, comme BMW, MAN, Adidas, Puma, Quelle et Allianz.</example>
+            <example>Si oui le quel et comment est-il venu chez nous en Belgique?</example>
+            <example>Quel es t la cotation de l 'avant derniere voie.</example>
+            <example>Une notation plus précise indique exactement quelles et combien de fonctions sont ajoutées à la base.</example>
+        </rule>
+        <rule default="temp_off">
+            <antipattern>
+                <token><exception postag="SENT_START"/></token>
+                <token regexp="yes">(?-i)Ne|(?-i)N|(?-i)N[’´'‛′‘]</token>
+                <token>et</token>
+                <token postag="A" min="0" max="1"/>
+                <token regexp="yes" inflected="yes">pas|plus|jamais|guère|que</token>
+            </antipattern>
+            <pattern>
+                <token inflected="yes">ne<exception scope="previous">=</exception></token>
+                <marker>
+                    <token>et</token>
+                </marker>
+                <token postag="A" min="0" max="1"/>
+                <token regexp="yes" inflected="yes">pas|plus|jamais|guère|que<exception scope="next" regexp="yes">[a-z]</exception></token>
+            </pattern>
+            <message>Un verbe est attendu après le marqueur de négation '\1'.</message>
+            <suggestion>est</suggestion>
+            <example correction="est">Il ne peut pas voler haut, car ce n'<marker>et</marker> encore qu'un bébé.</example>
+            <example>Il suffit que n = n et que les algèbre D et D soient isomorphes.</example>
+            <example>On suppose que E est de dimension finie non nulle n et que h est non dégénérée.</example>
+            <example>On déduit ( est équipotent à (AB)C), celle entre ({0,1}N)N et {0,1}N et donc que l'ensemble des suites réelles indexées par les entiers a la puissance du continu.</example>
+        </rule>
+        <rule default="temp_off">
+            <antipattern>
+                <token regexp="yes">(?-i)[MTS]'</token>
+            </antipattern>
+            <pattern>
+                <token regexp="yes">[mts][’´'‛′‘]<exception scope="previous">=</exception></token>
+                <marker>
```

Comment:
Don't put here all these apostrophe characters. It does nothing because wrong characters are tokenized separately. Use just `[mts]'`, which includes straight and curly apostrophe.